packages/playground/windows/playground/MainPage.cpp

```diff
@@ -28,6 +28,20 @@ MainPage::MainPage() {
     x_rootComponentNameCombo().IsEditable(true);
     x_entryPointCombo().IsEditable(true);
   }
+
+#if !defined(USE_HERMES)
+  x_engineHermes().IsEnabled(false);
+#endif
+
+#if !defined(USE_V8)
+  x_engineV8().IsEnabled(false);
+#endif
+
+#if !defined(USE_QUICKJS)
+  x_engineQuickJS().IsEnabled(false);
+#endif
```

Comment:
Is USE_QUICKJS ever defined by anyone?

Reply:
No, I got rid of it.

---
In reply to: [467268353](https://github.com/microsoft/react-native-windows/pull/5666#discussion_r467268353) [](ancestors = 467268353)